.github/workflows/reusable_benchmarks.yml

```diff
@@ -135,22 +140,21 @@ jobs:
 
     - name: Run UMF benchmarks
       id: benchmarks
-      working-directory: ${{env.BUILD_DIR}}
       run: >
-        source ${{github.workspace}}/.venv/bin/activate &&
-        taskset -c ${{ env.CORES }} ${{ github.workspace }}/sycl-repo/unified-runtime/scripts/benchmarks/main.py
+        source .venv/bin/activate &&
+        taskset -c ${{ env.CORES }} ./sc/devops/scripts/benchmarks/main.py
         ~/bench_workdir_umf
         --umf ${{env.BUILD_DIR}}
-        --compare baseline
         --timeout 3000
-        ${{ inputs.upload_report && '--output-html' || '' }}
-        ${{ inputs.pr_no != 0 && '--output-markdown' || '' }}
+        --output-html remote
+        --results-dir ${{ github.workspace }}/results-repo
+        --output-markdown
         ${{ inputs.bench_script_params }}
 
     # In case it failed to add a comment, we can still print the results.
     - name: Print benchmark results
-      if: ${{ always() && inputs.pr_no != 0 }}
-      run: cat ${{env.BUILD_DIR}}/benchmark_results.md
+      if: ${{ always() }}
```

Comment:
`${{}}` is not needed

Reply:
we use it everywhere in this workflow (in some cases there's `&&`) so I'd keep it for unity, I guess

Reply:
But it didn't run, does it work? https://github.com/oneapi-src/unified-memory-framework/actions/runs/13971793856/job/39115241536